t/data/flows/default_broken.yaml

```diff
@@ -0,0 +1,7 @@
+---
+# minimal default file with an intentionally broken 3rd step
+phase_1:
+  - step_1
+phase_2: []
+phase_3
```

Comment:
this is not a valid case, it is not valid yaml, the problem is that we don't check with yamllint the flows, we should ...